CharacterCreation/06_Skills.txt

```diff
@@ -65,6 +65,11 @@ this is not an exhaustive list of skills, merely the skills called for most
 often. Talk to your GM if you would like your character to learn a more specific
 case of one of these skills or a skill not listed below.
 
+	Assisted skills, which usually require an appropriate tool set as 
+described in the basic rules, are marked with an [A]. Keep in mind that, regardless 
+of [A]'s, having an appropriate set of tools for the task at hand will help with 
+almost any skill, though.

```

Comment:
We may want to make an explicit exception here for Reload checks. They're Weapons skill checks, but shouldn't require an entire skill check to perform.